setup.py

```diff
@@ -21,18 +24,25 @@
 with open('README.rst') as f:
     readme = f.read()
 
+PY3 = (sys.version_info[0] >= 3)
+
+install_requires = [
+    "pyOpenSSL >= 0.13",
+    "requests >= 2.1.0",
+    "service_identity >= 14.0.0",
+    "six"
+]
+
+if PY3:
```

Comment:
Because if you install a wheel, setup.py is t executed. 

I believe she did address it using the metadata changes in the other file but I can't test it atm.